src/openrct2/world/Wall.cpp

```diff
@@ -33,27 +33,27 @@
  *
  *  rct2: 0x006E588E
  */
-void wall_remove_at(int32_t x, int32_t y, int32_t z0, int32_t z1)
+void wall_remove_at(const CoordsXYRangedZ& wallPos)
 {
     TileElement* tileElement;
 
-    z0 /= 8;
-    z1 /= 8;
+    auto baseZ = wallPos.baseZ / 8;
```

Comment:
Remove all of the *8 and /8 and just use the tileElement->GetBaseZ() and such functions.